### DISCUSSION TITLE
trivy have not support  linux/armv6

### DISCUSSION THREAD

**Discussion by wangyimiao8603** [aquasecurity/trivy#4110] [Q&A]
1.download trivy_0.40.0_Linux-ARM.tar.gz from repo
2.# ./trivy
Illegal instruction

**Comment by knqyf263** [comment]
Interesting. It was reported that the ARM binary worked on armv6.
https://github.com/aquasecurity/trivy/pull/3985

**Reply by YOU54F** [reply]
I only tested against armv7 in that link, not armv6
